.env.example

```
@@ -124,6 +124,7 @@ SMP_REGISTRATION_MODE=CLOSED
 # ── Transactional emails ─────────────────────────────────────────────────────
 SMP_EMAIL_SENDER=noreply@profiletailors.com
 SMP_EMAIL_VERIFICATION_SUBJECT_PREFIX=[Profile Tailors]
+SMP_EMAIL_PUBLIC_APP_URL=https://app.profiletailors.com
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SMP_EMAIL_PUBLIC_APP_URL key should go before the SMP_EMAIL_SENDER key

(UnorderedKey)